docs/index.md

```diff
@@ -4,14 +4,32 @@ hide:
     - toc
 ---
 
-# Home
+# Browsertrix Docs
 
-Welcome to the Browsertrix official user guide and developer docs. These docs will contain the following sections.
+Browsertrix is an open source web archiving system created by [Webrecorder](https://webrecorder.net/). Sign up for Browsertrix on [browsertrix.com](https://browsertrix.com/) to start archiving with zero setup, or follow our self-hosting guide to deploy Browsertrix on your own infrastructure.
 
-- [Deployment Guide](deploy/index.md) — How to install and deploy Browsertrix on your local machine, or in the cloud.
-- [Developer Docs](develop/index.md) — Information on developing Browsertrix itself.
-- [User Guide](user-guide/index.md) — Instructions and reference for using Browsertrix.
+## Documentation
 
-If you are unfamiliar with Browsertrix, please check out [our website](https://browsertrix.com), or the main repository at [https://github.com/webrecorder/browsertrix](https://github.com/webrecorder/browsertrix)
+Browsertrix' Docs are organized into the following sections. If you have feedback on the docs, please feel free to [reach out to us](mailto:docs-feedback@webrecorder.net).
 
-If something is missing, unclear, or seems incorrect, please open an [issue](https://github.com/webrecorder/browsertrix/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) and we'll try to make sure that your questions get answered here in the future!
+### User Guide
+
+Instructions on how to create and share web archives [using the Browsertrix software](./user-guide/index.md).
```

Comment:
This link description text could be improved, especially for readers using assistive technology.  I think the previous format was better suited for this content than dedicated headings.  "Using the Browsertrix software" on its own is not a good descriptive link.

Reply:
Updated:

<img width="734" alt="Screenshot 2024-09-17 at 9 17 01 AM" src="https://github.com/user-attachments/assets/812b3e69-fce4-4cf7-842f-276fc696f572">